docker-compose.yml

```
@@ -0,0 +1,20 @@
+version: '3.8'
+services:
+  db:
+    build: .
+    container_name: raiseTech-course-task10
+    platform: linux/x86_64
+    command: --default-authentication-plugin=mysql_native_password
+    restart: always
+    environment:
+      MYSQL_ROOT_PASSWORD: developer
```

passwordがdeveloperというのにはちょっと違和感がある。
```suggestion
      MYSQL_ROOT_PASSWORD: password
```
でいい気がします。

---

実験的に変更して、戻すの忘れていました。
修正します！